From 1b98de71ede3ef468ab774d8d5e5ac618448d8f4 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 20 Apr 2016 14:00:21 +0800
Subject: [PATCH 1019/5242] MLK-12685 ARM: dts: imx6sx-sabreauto.dts: improve
 usdhc4 pad drive strength

commit  7c6ae5d26e8423673ad784950d07eb5945c04975 from
https://source.codeaurora.org/external/imx/linux-imx.git

For imx6sx-sabreauto board, the usdhc4 is used for the sd slot locate on the
base board, so need to improve the pad drive strength, otherwise we will meet
many CRC error or timeout error when insert a sd card.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index dc6e1c9..9d7e20a 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -988,14 +988,14 @@
 
 	pinctrl_usdhc4: usdhc4grp {
 		fsl,pins = <
-			MX6SX_PAD_SD4_CMD__USDHC4_CMD		0x17059
-			MX6SX_PAD_SD4_CLK__USDHC4_CLK		0x10059
-			MX6SX_PAD_SD4_DATA0__USDHC4_DATA0	0x17059
-			MX6SX_PAD_SD4_DATA1__USDHC4_DATA1	0x17059
-			MX6SX_PAD_SD4_DATA2__USDHC4_DATA2	0x17059
-			MX6SX_PAD_SD4_DATA3__USDHC4_DATA3	0x17059
-			MX6SX_PAD_SD4_DATA7__GPIO6_IO_21	0x17059 /* CD */
-			MX6SX_PAD_SD4_DATA6__GPIO6_IO_20	0x17059 /* WP */
+			MX6SX_PAD_SD4_CMD__USDHC4_CMD		0x17071
+			MX6SX_PAD_SD4_CLK__USDHC4_CLK		0x10071
+			MX6SX_PAD_SD4_DATA0__USDHC4_DATA0	0x17071
+			MX6SX_PAD_SD4_DATA1__USDHC4_DATA1	0x17071
+			MX6SX_PAD_SD4_DATA2__USDHC4_DATA2	0x17071
+			MX6SX_PAD_SD4_DATA3__USDHC4_DATA3	0x17071
+			MX6SX_PAD_SD4_DATA7__GPIO6_IO_21	0x17071 /* CD */
+			MX6SX_PAD_SD4_DATA6__GPIO6_IO_20	0x17071 /* WP */
 		>;
 	};
 
-- 
1.7.9.5

